### ISSUE TITLE
Raising `assert param.grad is not None` when finetuning LoRA.

### ISSUE THREAD

**Issue by HashimotoPatrickMu** [facebookresearch/fairscale#1188]
While fine-tuning LoRA using PyTorch Lightning, I consistently encounter an `assert param.grad is not None` error during backpropagation. Increasing the gradient accumulation steps delays the issue but doesn't resolve it. I suspect this might be a backend problem. Has anyone else experienced this issue?

**Comment by cao-hy23** [comment]
hi,i have the same question.Did you solve it?
